style.css

```diff
@@ -1,417 +1,621 @@
-  font-size: 12px;
-  text-align: center;
-  line-height: 16px;
-  letter-spacing: 0.03em;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
 }
 
-.see-project-btn {
-  padding: 12px;
-  background: #36b37f;
-  border-radius: 4px;
-  position: absolute;
-  width: 122px;
-  left: calc(50% - 122px / 2 + 0.5px);
-  bottom: 24px;
+.headline {
+  background: #fff url(./img/header-illsutration-mobile.svg) no-repeat;
+  background-size: 100%;
+  overflow: hidden;
+}
 
-  /* font-family: "Inter"; */
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  border: transparent;
-  color: #fff;
+.headline h1:first-child {
+  margin-top: 1.5em;
 }
 
-.divider {
-  background: #36b37f;
-  height: 4px;
-  border-radius: 5px;
-  width: 50px;
-  margin: 0 auto;
+.headline p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
 }
 
-.divider-container {
-  margin: 20px 0;
+.headline button {
+  background: none;
 }
 
-.about-myself {
+.social-media {
+  list-style: none;
+  padding: 0;
   display: flex;
-  flex-direction: column;
   justify-content: center;
-  align-items: center;
-  padding-top: 50px;
-  margin-bottom: 40px;
-  background-position: bottom;
-  background-size: cover;
-  z-index: 3;
-  position: relative;
+  width: 170px;
 }
 
-/*
-.about-myself {
-  background: url(./img/illustration\ 1-\ about\ me-mobile.svg), url(./img/illustration\ 2-about\ me\ mobile.svg);
-  background-repeat: no-repeat;
-  background-position: right top, left bottom;
+@media (min-width: 992px) {
+  .header-nav-phone {
+    display: none;
+  }
+
+  .header-title {
+    padding-top: 3em;
+  }
+
+  .header-nav-desktop {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    padding-top: 1em;
+    margin: 0;
+  }
+
+  .header-nav-desktop li {
+    list-style: none;
+    padding: 0;
+    margin: 1em;
+    font-size: 14px;
+    font-weight: bold;
+    color: #42526e;
+    cursor: pointer;
+  }
+
+  .headline {
+    background: #fff url(./img/Header-llustration-desktop.svg) no-repeat;
+    background-size: 100%;
+    padding-bottom: 8em;
+  }
 }
-*/
-.about-myself button {
-  background: #36b37f;
-  border: none;
-  font-size: 17px;
-  color: #fff;
-  padding: 12px;
-  border-radius: 4px;
-  gap: 8px;
+
+@media (min-width: 1024px) {
+  .headline {
+    padding-bottom: 15em;
+  }
 }
 
-.about-myself h1 {
-  margin-bottom: 68;
-  font-size: 40px;
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 200px repeat(5, 6fr);
   text-align: center;
-  color: #172b4d;
+  width: 90%;
+  margin: 0 auto;
 }
 
-.about-myself p {
-  text-align: center;
-  font-size: 20px;
-  color: #42526e;
-  font-style: normal;
-  line-height: 32px;
-  padding: 0 250px;
-  margin: 24px 12px;
-  font-weight: 300;
+.works h1:first-child {
+  padding: 1em;
 }
```

Comment:
- [ ] Please 🙏🏽 complete all the changes required by the previous code reviewer to approve the project. Kindly make sure that the mobile version looks similar to the Figma version for a better user interface.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/201496406-a9725d05-ac7d-42a4-be05-925c53a014ee.png) | ![image](https://user-images.githubusercontent.com/58642949/201496418-e69f649d-28ad-4a4d-82a0-d4707d85ad03.png) |